### PR TITLE
plasma-infra(audit): Resolve root dependencies(dev) vulnerabilities

### PR DESCRIPTION
**Pull request by Yakutoc**
## Release Notes

Почищен корневой package.json:

1. Удален storybook
2. Исправлен ряд critical vulnerabilities
3. Добавлены зависимости, которые раньше доставлялись транзитивно из-за установки storybook.      

### What/why Changed

**Storybook** удален так как является рудиментом от установки/использования библиотеки chromatic.    
 
Screenshot 2023-10-11 at 12 42 31

### Resolve transitive **dev dependencies**

- @babel/plugin-proposal-class-properties
- @babel/plugin-transform-react-constant-elements
- babel-loader
- url-loader

> Note: ранее доставлялись транзитивно из-за установки storybook и его плагинов

### Для корректного запуска perf test:

- react - для запуска perf test
- react-dom - для запуска perf test

## Resolve root dependencies(dev) vulnerabilities:

- https://github.com/salute-developers/plasma/security/dependabot/67
- https://github.com/salute-developers/plasma/security/dependabot/87
- https://github.com/salute-developers/plasma/security/dependabot/37
- https://github.com/salute-developers/plasma/security/dependabot/26

#### Before npm audit
Screenshot 2023-11-03 at 10 17 01

#### After npm audit
Screenshot 2023-11-03 at 11 32 36

#### Before dependabot alerts

before resolve root vul

--------

<h3>⚡ Component performance testing</h3>

**Result:** 🟢 OK

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>borderRadius</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_default_react-native--canary.824.6795562408.ts'>borderRadius_default_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_b2c_react-native--canary.824.6795562408.ts'>borderRadius_plasma_b2c_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_web_react-native--canary.824.6795562408.ts'>borderRadius_plasma_web_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberHealth_react-native--canary.824.6795562408.ts'>borderRadius_sberHealth_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_business_react-native--canary.824.6795562408.ts'>borderRadius_sbermarket_business_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_metro_react-native--canary.824.6795562408.ts'>borderRadius_sbermarket_metro_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_react-native--canary.824.6795562408.ts'>borderRadius_sbermarket_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_selgros_react-native--canary.824.6795562408.ts'>borderRadius_sbermarket_selgros_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_wlbusiness_react-native--canary.824.6795562408.ts'>borderRadius_sbermarket_wlbusiness_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberonline_react-native--canary.824.6795562408.ts'>borderRadius_sberonline_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberprime_react-native--canary.824.6795562408.ts'>borderRadius_sberprime_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_stylesSalute_react-native--canary.824.6795562408.ts'>borderRadius_stylesSalute_react-native--canary.824.6795562408.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_default_react-native--canary.824.6795562408.ts'>shadow_default_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.824.6795562408.ts'>shadow_plasma_b2c_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_web_react-native--canary.824.6795562408.ts'>shadow_plasma_web_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.824.6795562408.ts'>shadow_sberHealth_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.824.6795562408.ts'>shadow_sbermarket_business_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.824.6795562408.ts'>shadow_sbermarket_metro_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.824.6795562408.ts'>shadow_sbermarket_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.824.6795562408.ts'>shadow_sbermarket_selgros_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.824.6795562408.ts'>shadow_sbermarket_wlbusiness_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.824.6795562408.ts'>shadow_sberonline_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.824.6795562408.ts'>shadow_sberprime_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.824.6795562408.ts'>shadow_stylesSalute_react-native--canary.824.6795562408.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_ios-swift--canary.824.6795562408.swift'>color_default_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_kotlin--canary.824.6795562408.kt'>color_default_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_react-native--canary.824.6795562408.ts'>color_default_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_xml--canary.824.6795562408.xml'>color_default_xml--canary.824.6795562408.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.824.6795562408.swift'>color_plasma_b2c_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.824.6795562408.kt'>color_plasma_b2c_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.824.6795562408.ts'>color_plasma_b2c_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.824.6795562408.xml'>color_plasma_b2c_xml--canary.824.6795562408.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_ios-swift--canary.824.6795562408.swift'>color_plasma_web_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_kotlin--canary.824.6795562408.kt'>color_plasma_web_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_react-native--canary.824.6795562408.ts'>color_plasma_web_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_xml--canary.824.6795562408.xml'>color_plasma_web_xml--canary.824.6795562408.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.824.6795562408.swift'>color_sberHealth_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.824.6795562408.kt'>color_sberHealth_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.824.6795562408.ts'>color_sberHealth_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.824.6795562408.xml'>color_sberHealth_xml--canary.824.6795562408.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.824.6795562408.swift'>color_sbermarket_business_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.824.6795562408.kt'>color_sbermarket_business_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.824.6795562408.ts'>color_sbermarket_business_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.824.6795562408.xml'>color_sbermarket_business_xml--canary.824.6795562408.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.824.6795562408.swift'>color_sbermarket_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.824.6795562408.kt'>color_sbermarket_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.824.6795562408.swift'>color_sbermarket_metro_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.824.6795562408.kt'>color_sbermarket_metro_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.824.6795562408.ts'>color_sbermarket_metro_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.824.6795562408.xml'>color_sbermarket_metro_xml--canary.824.6795562408.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.824.6795562408.ts'>color_sbermarket_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.824.6795562408.swift'>color_sbermarket_selgros_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.824.6795562408.kt'>color_sbermarket_selgros_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.824.6795562408.ts'>color_sbermarket_selgros_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.824.6795562408.xml'>color_sbermarket_selgros_xml--canary.824.6795562408.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.824.6795562408.swift'>color_sbermarket_wlbusiness_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.824.6795562408.kt'>color_sbermarket_wlbusiness_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.824.6795562408.ts'>color_sbermarket_wlbusiness_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.824.6795562408.xml'>color_sbermarket_wlbusiness_xml--canary.824.6795562408.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.824.6795562408.xml'>color_sbermarket_xml--canary.824.6795562408.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.824.6795562408.swift'>color_sberonline_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.824.6795562408.kt'>color_sberonline_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.824.6795562408.ts'>color_sberonline_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.824.6795562408.xml'>color_sberonline_xml--canary.824.6795562408.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.824.6795562408.swift'>color_sberprime_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.824.6795562408.kt'>color_sberprime_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.824.6795562408.ts'>color_sberprime_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.824.6795562408.xml'>color_sberprime_xml--canary.824.6795562408.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.824.6795562408.swift'>color_stylesSalute_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.824.6795562408.kt'>color_stylesSalute_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.824.6795562408.ts'>color_stylesSalute_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.824.6795562408.xml'>color_stylesSalute_xml--canary.824.6795562408.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>spacing</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_default_react-native--canary.824.6795562408.ts'>spacing_default_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_b2c_react-native--canary.824.6795562408.ts'>spacing_plasma_b2c_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_web_react-native--canary.824.6795562408.ts'>spacing_plasma_web_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberHealth_react-native--canary.824.6795562408.ts'>spacing_sberHealth_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_business_react-native--canary.824.6795562408.ts'>spacing_sbermarket_business_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_metro_react-native--canary.824.6795562408.ts'>spacing_sbermarket_metro_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_react-native--canary.824.6795562408.ts'>spacing_sbermarket_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_selgros_react-native--canary.824.6795562408.ts'>spacing_sbermarket_selgros_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_wlbusiness_react-native--canary.824.6795562408.ts'>spacing_sbermarket_wlbusiness_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberonline_react-native--canary.824.6795562408.ts'>spacing_sberonline_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberprime_react-native--canary.824.6795562408.ts'>spacing_sberprime_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_stylesSalute_react-native--canary.824.6795562408.ts'>spacing_stylesSalute_react-native--canary.824.6795562408.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.824.6795562408.swift'>typo_mage_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.824.6795562408.kt'>typo_mage_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.824.6795562408.ts'>typo_mage_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.824.6795562408.swift'>typo_plasma_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.824.6795562408.kt'>typo_plasma_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.824.6795562408.ts'>typo_plasma_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.824.6795562408.swift'>typo_ruler_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.824.6795562408.kt'>typo_ruler_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.824.6795562408.ts'>typo_ruler_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.824.6795562408.swift'>typo_sage_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.824.6795562408.kt'>typo_sage_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.824.6795562408.ts'>typo_sage_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.824.6795562408.swift'>typo_sbermarket_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.824.6795562408.kt'>typo_sbermarket_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.824.6795562408.ts'>typo_sbermarket_react-native--canary.824.6795562408.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.824.6795562408.swift'>typo_soulmate_ios-swift--canary.824.6795562408.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.824.6795562408.kt'>typo_soulmate_kotlin--canary.824.6795562408.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.824.6795562408.ts'>typo_soulmate_react-native--canary.824.6795562408.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-824/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-824/
ui storybook: http://plasma.sberdevices.ru/pr/pr-824/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-824/temple-storybook/

**Review by LamaEats — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-824/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-824/
ui storybook: http://plasma.sberdevices.ru/pr/pr-824/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-824/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-824/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-824/
ui storybook: http://plasma.sberdevices.ru/pr/pr-824/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-824/temple-storybook/

File: packages/plasma-ui/package.json [hunk 1]
```diff
@@ -1,116 +1,113 @@
...
-    "test:watch": "BABEL_ENV=cjs jest --watch",
-    "typescript-coverage": "npx typescript-coverage-report > /dev/null",
-    "lint": "../../node_modules/.bin/eslint ./src --ext .js,.ts,.tsx --quiet"
-  },
-  "typeCoverage": {
-    "ignoreFiles": [
-      "src/**/*component-test.tsx",
-      "src/**/*stories.tsx",
-      "src/**/*perftest.tsx"
+    "name": "@salutejs/plasma-ui",
```
Comment (Yakutoc) [inline packages/plasma-ui/package.json:2]: @neretin-trike @Yeti-or  Я правильно понимаю, что это правильный размер отступов.

File: auto-plugins/tsconfig.json [hunk 2]
```diff
@@ -9,5 +9,5 @@
         "outDir": "./dist"
     },
     "include": ["./src"],
-    "exclude": ["node_modules/*"]
```
Comment (neretin-trike) [inline auto-plugins/tsconfig.json:12]: а почему сюда переехало?

**Review by neretin-trike — APPROVED** [review]

File: auto-plugins/tsconfig.json (hunk 2 above)
Comment (Yakutoc) [reply auto-plugins/tsconfig.json:12]: @neretin-trike 

А мы и так выключили этот плагин "update-deps". 

И что бы он не собирался, и не требовал зависимости - https://github.com/salute-developers/plasma/pull/824/commits/4df6ac8055715c4ece72890a88672158c6a7e30a 

Я решил его исключить из сборки.

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-824/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-824/
ui storybook: http://plasma.sberdevices.ru/pr/pr-824/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-824/temple-storybook/

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.242.0-dev.0, @salutejs/plasma-hope@1.240.0-dev.0, @salutejs/plasma-icons@1.169.0-dev.0, @salutejs/plasma-temple-docs@0.179.0-dev.0, @salutejs/plasma-temple@1.187.0-dev.0, @salutejs/plasma-ui-docs@0.228.0-dev.0, @salutejs/plasma-ui@1.219.0-dev.0, @salutejs/plasma-web-docs@0.196.0-dev.0, @salutejs/plasma-web@1.242.0-dev.0, @salutejs/plasma-website@0.204.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.242.0-dev.0, @salutejs/plasma-hope@1.240.0-dev.0, @salutejs/plasma-icons@1.169.0-dev.0, @salutejs/plasma-temple-docs@0.179.0-dev.0, @salutejs/plasma-temple@1.187.0-dev.0, @salutejs/plasma-ui-docs@0.228.0-dev.0, @salutejs/plasma-ui@1.219.0-dev.0, @salutejs/plasma-web-docs@0.196.0-dev.0, @salutejs/plasma-web@1.242.0-dev.0, @salutejs/plasma-website@0.204.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.242.0-dev.0, @salutejs/plasma-hope@1.240.0-dev.0, @salutejs/plasma-icons@1.169.0-dev.0, @salutejs/plasma-temple-docs@0.179.0-dev.0, @salutejs/plasma-temple@1.187.0-dev.0, @salutejs/plasma-ui-docs@0.228.0-dev.0, @salutejs/plasma-ui@1.219.0-dev.0, @salutejs/plasma-web-docs@0.196.0-dev.0, @salutejs/plasma-web@1.242.0-dev.0, @salutejs/plasma-website@0.204.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.242.0-dev.0, @salutejs/plasma-hope@1.240.0-dev.0, @salutejs/plasma-icons@1.169.0-dev.0, @salutejs/plasma-temple-docs@0.179.0-dev.0, @salutejs/plasma-temple@1.187.0-dev.0, @salutejs/plasma-ui-docs@0.228.0-dev.0, @salutejs/plasma-ui@1.219.0-dev.0, @salutejs/plasma-web-docs@0.196.0-dev.0, @salutejs/plasma-web@1.242.0-dev.0, @salutejs/plasma-website@0.204.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.242.0-dev.0, @salutejs/plasma-hope@1.240.0-dev.0, @salutejs/plasma-icons@1.169.0-dev.0, @salutejs/plasma-temple-docs@0.179.0-dev.0, @salutejs/plasma-temple@1.187.0-dev.0, @salutejs/plasma-ui-docs@0.228.0-dev.0, @salutejs/plasma-ui@1.219.0-dev.0, @salutejs/plasma-web-docs@0.196.0-dev.0, @salutejs/plasma-web@1.242.0-dev.0, @salutejs/plasma-website@0.204.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.242.0-dev.0, @salutejs/plasma-hope@1.240.0-dev.0, @salutejs/plasma-icons@1.169.0-dev.0, @salutejs/plasma-temple-docs@0.179.0-dev.0, @salutejs/plasma-temple@1.187.0-dev.0, @salutejs/plasma-ui-docs@0.228.0-dev.0, @salutejs/plasma-ui@1.219.0-dev.0, @salutejs/plasma-web-docs@0.196.0-dev.0, @salutejs/plasma-web@1.242.0-dev.0, @salutejs/plasma-website@0.204.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-b2c@1.242.0-dev.0, @salutejs/plasma-hope@1.240.0-dev.0, @salutejs/plasma-icons@1.169.0-dev.0, @salutejs/plasma-temple-docs@0.179.0-dev.0, @salutejs/plasma-temple@1.187.0-dev.0, @salutejs/plasma-ui-docs@0.228.0-dev.0, @salutejs/plasma-ui@1.219.0-dev.0, @salutejs/plasma-web-docs@0.196.0-dev.0, @salutejs/plasma-web@1.242.0-dev.0, @salutejs/plasma-website@0.204.0-dev.0 🚀

**Pull request merged**